src/control/dmg/main.go

```diff
@@ -48,19 +45,18 @@ type cliOptions struct {
 }
 
 var (
-	opts  = new(cliOptions)
-	conns = client.NewConnect()
+	opts   = new(cliOptions)
+	conns  = client.NewConnect()
+	config = client.NewConfiguration()
 )
 
 func connectHosts() error {
+
 	if opts.Hostfile != "" {
 		return errors.New("hostfile option not implemented")
 	}
-	hosts := strings.Split(opts.Hostlist, ",")
```

Comment:
why change this Logic?

Reply:
The host list processing was moved out of connectHosts and into applyCmdLineOverrides because the final host list is that stored in config.HostList instead of opts.Hostlist.  applyCmdLineOverrides now implements the string split opts.Hostlist is provided.

Reply:
yes but you've removed the check for len < 1

Reply:
Had conversation with Tom offline, but for the purposes of having it in the record, the check for "len < 1" was removed because it is no longer necessary.  The default configuration provides a non empty HostList.  Overrides from the config file or the commandline are only made if they are not empty.